pages/resources.js

```diff
@@ -1,15 +1,160 @@
+import Link from "next/link";
+import { useState } from "react";
 import styled from "styled-components";
 
 const Resources = () => {
   return (
     <div>
       <Headline>Hilfestellen und Beratungsangebote</Headline>
       <List>
-        <ListItem>Fachberatung1</ListItem>
-        <ListItem>Fachberatung2</ListItem>
-        <ListItem>Fachberatung3</ListItem>
-        <ListItem>Fachberatung4</ListItem>
-        <ListItem>Fachberatung5</ListItem>
+        <ListItem>
+          <h3>bff: Gewalt gegen Frauen e.V.</h3>
+          <Link href="https://www.frauen-gegen-gewalt.de" passHref>
+            bff: Gewalt gegen Frauen e.V.
+          </Link>
+        </ListItem>
+        <ListItem>
+          <h3>Hilfetelefon Gewalt gegen Frauen</h3>
+          <Link href="https://www.hilfetelefon.de/" passHref>
+            Hilfetelefon Gewalt gegen Frauen
+          </Link>
+          <br />
```

Comment:
rather work with margins, paddings or flex+gap. Breaks are the worst method to generate spaces in your stylings